dev/tools/gen_keycodes/data/keyboard_key.tmpl

```diff
@@ -58,7 +58,7 @@ abstract class KeyboardKey with Diagnosticable {
 /// // The node used to request the keyboard focus.
 /// final FocusNode _focusNode = FocusNode();
 /// // The message to display.
-/// String _message;
+/// String? _message;
```

Comment:
Nit: Could this be `late` instead of nullable?  Below it looks like it can never be assigned to null.  I'm not sure which one is intended to be used in situations like this.

Reply:
Ditto on the other `_message` variables below.

Reply:
Near the bottom of the template there is a line: 

```dart
          return Text(_message ?? 'Press a key');
```

Which assumes that it can be null.

 I'll take another look at the others and see if they should be late instead. Thx.

Reply:
Ah I missed that.  Sounds good either way.